shanghai/logging.py

```diff
@@ -24,9 +25,14 @@ def set_logging_config(config):
 
 def _print_like(func):
     @functools.wraps(func)
-    def _wrap(self, *args):
+    def _wrap(self, *args, **kwargs):
         f = io.StringIO()
         print(*args, file=f)
+        if 'exc_info' in kwargs:
+            exc_info = kwargs.pop('exc_info')
+            if exc_info:
+                traceback.print_exc(file=f)
```

Comment:
Are you sure this does what you think it does? I mean, `traceback.print_exec` takes the currently being handled exception from `sys` iirc, so passing `exec_info` doesn't mean anything here. In fact, I'm quite sure this is what `.exception` already does, only at the exception level and not at any other (which I don't think we need?).